### PR TITLE
closed #60 & #61

### LINKED ISSUE #60: Add information to about page
Please design the about page following the design of https://www.google.com/intl/en/about/

Please add the following information
- [ ] one column
  Our mission is to make the world’s social media information openly accessible and useful generating open knowledge for all. # OpenData (/search?query=#OpenData) #OpenSource (/search?query=#OpenSource) #OpenKnowledge (/search?query=#OpenKnowledge)
- [ ] three columns

News from us
Read up on our latest news and browse our blog (http://blog.loklak.net) for updates on our work. 

Developers
Wanna know how to set up loklak yourself or you would like to develop an app based on loklak. On http://api.loklak.org you find more info about our APIs.

Contribute to our projects
Ever wondered what life as a developers looks like or what benefits you might expect as an Open Source contributor? Search for an issue to solve on our repositories with FOSSASIA (https://github.com/fossasia?utf8=%E2%9C%93&query=loklak) or the loklak org (http://github.com/loklak/) and check out feature requests and issues. 
- [ ] adjust height and use this image
  nasa-apollo8-dec24-earthrise

### PR DESCRIPTION
**Pull request by rishiraj824**
@mariobehling Please check the commit for the issues #60 & #61 as assigned. Thank You.

### CODE REVIEW

**Comment by mariobehling** [comment]
Please ensure build tests are passing. Travis will pass then.

**Comment by hemantjadon** [comment]
@rishiraj824 errors are due to you don't have the latest code in your fork.
Please check :)

**Comment by rishiraj824** [comment]
Okay will check. Thanks.

**Comment by hemantjadon** [comment]
@rishiraj824 also for quick test you can run `ng test` and `ng lint` locally

**Comment by rishiraj824** [comment]
@mariobehling Please check the latest commit on the master branch. There shouldn't be any errors now.
this screenshot is of ng lint and ng test
screenshot from 2016-10-23 16 47 02
screenshot from 2016-10-23 16 09 35
This is the About Page #60 
screenshot from 2016-10-23 16 08 59
This is the contact page #61 
screenshot from 2016-10-23 16 54 27

**Comment by rishiraj824** [comment]
The errors in travis is from the home and the feed components.

**Comment by mariobehling** [comment]
@hemantjadon Please help to resolve merge conflicts.

@rishiraj824 Please provide a link to a test site.

There is some text in red/pinkish color. These are supposed to link to internal search results e.g. #OpenData -- links to --> /search?query=#OpenData

File: src/app/about/about.component.html [hunk 1]
```diff
@@ -1 +1,92 @@
...
+      </div>
+      <hr>
+      <h3>Developers</h3>
+      <p>Wanna know how to set up loklak yourself or you would like to develop an app based on loklak. On <a href="http://api.loklak.org" target="_blank">http://api.loklak.org</a> you find more info about our APIs.</p>
+      <h3>Contribute to our projects</h3>
+      <p>Ever wondered what life as a developers looks like or what benefits you might expect as an Open Source contributor? Search for an issue to solve on our repositories with <a href="https://github.com/fossasia?utf8=%E2%9C%93&query=loklak" target="_blank">FOSSASIA (https://github.com/fossasia?utf8=%E2%9C%93&query=loklak)</a> or the <a href="http://github.com/loklak/" target="_blank">loklak org (http://github.com/loklak/)</a> and check out feature requests and issues.</p>
+   </div>
+</div>
+<app-footer></app-footer>
+<style type="text/css">
```
Comment (hemantjadon) [inline src/app/about/about.component.html]: Move these To about.component.scss

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			</address>			
+			<hr>
+			<h5>Report a safety or abuse issue affecting our products.<br><br>
+			If you know of a safety or abuse problem with any of Google's services, we'd like to hear about it right away. Please use our contact form to report the issue.</h5>
+		</div>
+	</div>
+</div>
+<app-footer></app-footer>
+
+<style type="text/css">
```
Comment (hemantjadon) [inline src/app/contact/contact.component.html]: move these to `contact.component.scss`

**Comment by hemantjadon** [comment]
@rishiraj824 
I fixed these `linting errors` in last commit of mine.
Please safely pull that commit: 

* Make a branch `<branch-name>` so that all your changes are safe.

* Then refer to this https://help.github.com/articles/syncing-a-fork/ for syncing your master branch with the `main repo`

* merge `<branch-name>` to `master` locally, reslove conflicts if any.

* push the changes.

These should resolve all the `linting` errors

I also want you to please move the styles to `<component-name>.component.scss`, rather directly after `html` file for uniformity in code base.

Cheers :)

**Comment by mariobehling** [comment]
@rishiraj824 Any questions regarding this issue at the moment?

**Comment by rishiraj824** [comment]
@mariobehling i fixed the issues #60 & #61 already. Please check my last commit. I thought the changes were merged.

**Comment by rishiraj824** [comment]
screenshot from 2016-10-27 21 01 13
ng lint test on my local

**Comment by mariobehling** [comment]
Ok, closing it. I am following up about more details in the related issue.

**Pull request closed without merging**